Добавлена работа с документами

dartusha opened the pull request.

dartusha committed:
Добавлена работа с документами

zayarniy-alex commented on src/main/java/ru/geekbrains/entities/DocFilter.java:
```diff
@@ -0,0 +1,55 @@
+package ru.geekbrains.entities;
+
```
@Entity
@Data
@NoArgsConstructor

zayarniy-alex commented on src/main/java/ru/geekbrains/entities/DocFilter.java:
```diff
@@ -0,0 +1,55 @@
+
+    public Long getIdObject() {
+        return idObject;
+    }
+
+    public void setIdObject(Long idObject) {
+        this.idObject = idObject;
+    }
+
+    public String getTypeObject() {
+        return typeObject;
+    }
+
+    public void setTypeObject(String typeObject) {
+        this.typeObject = typeObject;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
```
Это всё не нужно, т.к. входит в состав @Data

dartusha committed:
1. Исправлено замечание по PR13 - убраны геттеры/сеттеры по DocFilter
2. Исправлены названия кнопок
3. Цветовая схема кнопок изменена, чтобы быть похожей на Tasks
